docs/api/data-models/service-lookup-request.md

```diff
@@ -0,0 +1,14 @@
+# ServiceLookupRequest
+
+Field | Type | Mandatory | Description
+--- | --- | --- | ---
+verbose | [Boolean](../primitives.md#boolean) | no | If true detailed system and device information also returns (only if the provider supports it).
```

Comment:
in IDDs, we basically describe the JSONs. That why we omit the authentication (which is a header) and status (which is the HTTP status code). And that's why verbose shouldn't be part of this model (it is a query parameter). Verbose should be describe in the operation description.